services/minespace-web/src/components/pages/Project/InformationRequirementsTablePage.js

```diff
@@ -0,0 +1,193 @@
+import IRTFileImport from "../../Forms/projects/informationRequirementsTable/IRTFileImport";
+
+const propTypes = {
+  project: CustomPropTypes.project.isRequired,
+  fetchProjectById: PropTypes.func.isRequired,
+  clearInformationRequirementsTable: PropTypes.func.isRequired,
+  match: PropTypes.shape({
+    params: {
+      mineGuid: PropTypes.string,
+    },
+  }).isRequired,
+};
+
+const StepForms = (props, state, next, prev, handleIRTSubmit) => [
+  {
+    title: "Download template",
+    content: <IRTDownloadTemplate />,
+    buttons: [
+      null,
+      <Button id="step1-next" type="tertiary" onClick={() => next()}>
+        Next
+      </Button>,
+    ],
+  },
+  {
+    title: "Import",
+    content: <IRTFileImport projectGuid={props.project.project_guid} />,
+    buttons: [
+      <Button
+        id="step-back"
+        style={{ display: "inline", float: "left" }}
+        type="tertiary"
+        className="full-mobile"
+        onClick={() => prev()}
+        disabled={state.submitting}
+      >
+        Back
+      </Button>,
+      <Button
+        id="step2-next"
+        style={{ display: "inline", float: "right" }}
+        type="tertiary"
+        onClick={() => next()}
+        disabled={state.submitting}
+      >
+        Next
+      </Button>,
+    ],
+  },
+  {
+    title: "Review / Submit",
+    content: (
+      <>
+        <br />
+      </>
+    ),
+    buttons: [
+      <Button
+        id="step-back"
+        style={{ display: "inline", float: "left" }}
+        type="tertiary"
+        className="full-mobile"
+        onClick={() => prev()}
+        disabled={state.submitting}
+      >
+        Back
+      </Button>,
+      <Button
+        type="primary"
+        style={{ display: "inline", float: "right" }}
+        htmlType="submit"
+        onClick={(event) => handleIRTSubmit(event)}
+        disabled={state.submitting}
+      >
+        Submit final IRT
+      </Button>,
+    ],
+  },
+];
+
+export class InformationRequirementsTablePage extends Component {
+  state = {
+    current: 0,
+    submitting: false,
+    isLoaded: false,
+    isEditMode: false,
+  };
+
+  componentDidMount() {
+    this.handleFetchData();
+  }
+
+  componentWillUnmount() {
+    this.props.clearInformationRequirementsTable();
+  }
+
+  next = () => this.setState((prevState) => ({ current: prevState.current + 1 }));
+
+  prev = () => this.setState((prevState) => ({ current: prevState.current - 1 }));
+
+  handleFetchData = () => {
+    const { projectGuid, tab } = this.props.match?.params;
+    if (projectGuid) {
+      return this.props
+        .fetchProjectById(projectGuid)
+        .then(() => this.setState({ isLoaded: true, activeTab: tab }));
+    }
+    return this.props.fetchProjectById(projectGuid).then(() => {
+      this.setState({ isLoaded: true, activeTab: tab });
+    });
+  };
+
+  render() {
+    const title = this.state.isEditMode
```

Comment:
This title is referencing the project summary title, I think we want it to reference the project title right?

Reply:
Good catch, changed.